src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -44,6 +45,7 @@ const initialState = {
   confirmed: null,
   confirmedStatus: FETCH_STATUS.notStarted,
   past: null,
+  providerData: null,
```

Comment:
I like the idea of having 'providerData' outside of 'appointmentDetails' in the event we ever try to normalize the Redux store.